core/build.gradle

```diff
@@ -91,11 +91,11 @@ dependencies {
     implementation "org.jooq:jooq:${jooqVersion}"
     implementation "software.amazon.awssdk:applicationautoscaling:${awssdkVersion}"
     implementation "software.amazon.awssdk:dynamodb:${awssdkVersion}"
-    implementation "software.amazon.awssdk:core:${awssdkVersion}"
+    implementation "software.amazon.awssdk:core:${awssdkVersion}@pom"
```

Comment:
Specified `pom` type explicitly for the `awssdk` dependency. This will fix the dependency error.